models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
```

Comment:
please remove this file

Reply:
reading on how to remove files from a pull request. what's the easiest way?

Reply:
just remove the file and commit that, the pull request should pick up the changes